data/migrations/v1.0.0-point.sql

```diff
@@ -8,7 +8,8 @@ UPDATE
      tags -> 'amenity' = 'grave_yard' OR
      tags -> 'boundary' IN ('national_park', 'protected_area') OR
      tags -> 'power' IN ('plant', 'substation') OR
-     tags -> 'natural' IN ('wood', 'forest'))
+     tags -> 'natural' IN ('wood', 'forest') OR
+     tags -> 'tourism' = 'gallery')
```

Comment:
Let's ensure that the polygon table also includes this `mz_poi_min_zoom` migration. Generally speaking, the `mz_poi_min_zoom` updates in the pois table should also be reflected in the polygon table. I forgot about this when I added this migration in the first place.